### PR TITLE
Simplify bug report template

### PR DESCRIPTION
**Pull request by ethanwharris**
## What does this PR do?

- simplifies the bug report template
- adds back the script to run to get environment details

## Before submitting

- [x] Was this **discussed/approved** via a GitHub issue? (not for typos and docs)
- [x] Did you read the [contributor guideline](https://github.com/Lightning-AI/lightning/blob/master/.github/CONTRIBUTING.md), **Pull Request** section?
- [x] Did you make sure your **PR does only one thing**, instead of bundling different changes together?
- [x] Did you make sure to **update the documentation** with your changes? (if necessary)
- [ ] Did you write any **new necessary tests**? (not for typos and docs)
- [ ] Did you verify new and **existing tests pass** locally with your changes?
- [ ] Did you list all the **breaking changes** introduced by this pull request?
- [ ] Did you **update the CHANGELOG**? (not for typos, docs, test updates, or minor internal changes/refactors)

## PR review

Anyone in the community is welcome to review the PR.
Before you start reviewing, make sure you have read the [review guidelines](https://github.com/Lightning-AI/lightning/wiki/Review-guidelines). In short, see the following bullet-list:

- [x] Is this pull request ready for review? (if not, please submit in draft mode)
- [ ] Check that all items from **Before submitting** are resolved
- [ ] Make sure the title is self-explanatory and the description concisely explains the PR
- [ ] Add labels and milestones (and optionally projects) to the PR so it can be classified

## Did you have fun?

Make sure you had fun coding 🙃

cc @borda @rohitgr7 @Felonious-Spellfire

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/1_bug_report.yaml [hunk 1]
```diff
@@ -1,43 +1,18 @@
 name: Report a bug
 description: Any errors that are not UI related.
-labels: ["needs triage", "bug"]
```
Comment (rohitgr7) [inline .github/ISSUE_TEMPLATE/1_bug_report.yaml:3]: why did you remove this?

File: .github/ISSUE_TEMPLATE/1_bug_report.yaml [hunk 2]
```diff
@@ -65,9 +40,18 @@ body:
...
-      label: Important info
-      description: >
-        Provide all the following info
+      label: Environment
+      description: |
+        Please provide details about your enviornment by copying and pasting the output from our environment collection script:
+        https://raw.githubusercontent.com/Lightning-AI/lightning/master/requirements/collect_env_details.py
+        (For security purposes, please check the contents of the script before running it)
+        You can get the script and run it with:
+        ```bash
```
Comment (rohitgr7) [inline .github/ISSUE_TEMPLATE/1_bug_report.yaml:49]:
```suggestion
        ```console
```
not 100% sure but I think console gives an option to copy the command in the UI.

**Review by rohitgr7 — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/1_bug_report.yaml (hunk 1 above)
Comment (ethanwharris) [reply .github/ISSUE_TEMPLATE/1_bug_report.yaml:3]: Suggestion from @carmocca here: https://github.com/Lightning-AI/lightning/pull/14193#discussion_r972920380

File: .github/ISSUE_TEMPLATE/1_bug_report.yaml [hunk 3]
```diff
@@ -1,43 +1,18 @@
 name: Report a bug
 description: Any errors that are not UI related.
-labels: ["needs triage", "bug"]
+labels: ["needs triage"]
 body:
-  - type: markdown
```
Comment (edenlightning) [inline .github/ISSUE_TEMPLATE/1_bug_report.yaml:5]: There's still value in instructing users to 
1. See if issue already exist
2. Read the docs
3. Ask questions in slack

File: .github/ISSUE_TEMPLATE/1_bug_report.yaml [hunk 4]
```diff
@@ -1,43 +1,18 @@
...
-        - label: I've added a code sample.
-          required: true
-        - label: I've provided any other important info that is required.
-          required: true
-
   - type: textarea
     attributes:
       label: Bug description
-      description: A clear and concise description of the 🐛bug🐛.
+      description: A description of the 🐛bug🐛.
```
Comment (edenlightning) [inline .github/ISSUE_TEMPLATE/1_bug_report.yaml:8]: Maybe we can add a placeholder?
```suggestion
      description: A description of the 🐛bug🐛.
    placeholder: |
      A clear and concise description of what the bug is.
      
      Include steps to reproduce, the expected behavior, and the actual behavior.
      
       ```
      The error message you got, with the full traceback if available.
      ```      
```

File: .github/ISSUE_TEMPLATE/1_bug_report.yaml [hunk 5]
```diff
@@ -46,7 +21,7 @@ body:
       placeholder: "# Insert code here"
```
Comment (edenlightning) [inline .github/ISSUE_TEMPLATE/1_bug_report.yaml:21]:
```suggestion
    placeholder: |
      ```python
      # Sample code to reproduce the problem
      ```
```

**Review by edenlightning — APPROVED** [review]

**Review by tchaton — APPROVED** [review]
LGTM !

**Review by carmocca — APPROVED** [review]
Thank you!

File: .github/ISSUE_TEMPLATE/1_bug_report.yaml (hunk 3 above)
Comment (ethanwharris) [reply .github/ISSUE_TEMPLATE/1_bug_report.yaml:5]: Sure, but the page where you choose what kind of issue you have already has links to our slack and docs - so if we have it here too then you will see it twice before being able to report your bug
